packages/govuk-frontend/src/govuk/components/checkboxes/checkboxes.mjs

```diff
@@ -51,13 +51,21 @@ export class Checkboxes extends GOVUKFrontendComponent {
 
     this.$inputs.forEach(($input) => {
       const targetId = $input.getAttribute('data-aria-controls')
```

Comment:
We create item and conditional IDs internally in the template: 

https://github.com/alphagov/govuk-frontend/blob/704cf6d1a29a5478a25542575a2197ebaf5e1c66/packages/govuk-frontend/src/govuk/components/checkboxes/template.njk#L62

However, this check is still useful for folks using HTML instead of the Nunjucks macros, as they'd be inputting their own `data-aria-controls` and ids.